src/wrapped/wrappedlibv4l2.c

```diff
@@ -0,0 +1,65 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#define _GNU_SOURCE         /* See feature_test_macros(7) */
+#include <dlfcn.h>
+
+#include "wrappedlibs.h"
+
+#include "wrapper.h"
+#include "bridge.h"
+#include "librarian/library_private.h"
+#include "x86emu.h"
+
+const char* libv4l2Name = "libv4l2.so.0";
+#define LIBNAME libv4l2
+
+#include "wrappedlib_init.h"
+
+typedef int     (*iFpi_t)           (void*, int);
+typedef int     (*iFi_t)            (int);
+typedef int     (*iFiii_t)          (int, int, int);
+typedef int     (*iFipu_t)          (int, void*, uint32_t);
+typedef void*   (*pFpuiiiI_t)       (void*, uint32_t, int, int, int, int64_t);
+typedef int     (*iFpu_t)           (void*, uint32_t);
+
+static library_t* my_lib = NULL;
+
+#define SUPER() \
+    GO(v4l2_open, iFpi_t)       \
+    GO(v4l2_close, iFi_t)       \
+    GO(v4l2_ioctl, iFiii_t)     \
+    GO(v4l2_read, iFipu_t)      \
+    GO(v4l2_mmap, pFpuiiiI_t)   \
+    GO(v4l2_munmap, iFpu_t)
+
```

Comment:
This part is only needed when the function use a Callback or some structure that need re-aligment, or anything else that require to create a "my_" function.
Fo automatic function (the simple "GO"), this is not needed.